File: C#/uri-1006.cs
// URI - 1006

using System;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            double noteA = double.Parse(Console.ReadLine());
            double noteB = double.Parse(Console.ReadLine());
            double noteC = double.Parse(Console.ReadLine());

            Console.WriteLine("MEDIA = " + ((noteA * 2.0 + noteB * 3.0 + noteC * 5.0) / 10).ToString("F1"));
        }
    }
}

File: Node/uri-1006.js
// URI - 1006

const readline = require('readline')

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
})

rl.on('line', input => {
    const noteA = parseFloat(input)

    rl.on('line', input => {
        const noteB = parseFloat(input)

        rl.on('line', input => {
            const noteC = parseFloat(input)
            const averageGrade = ((noteA * 2.0 + noteB * 3.0 + noteC * 5.0) / 10.0)

            console.log('MEDIA = ' + averageGrade.toFixed(1))

            rl.close()
        })
    })
})